flask_common/mongo/fields/phone.py

```diff
@@ -76,7 +79,7 @@ def to_local_formatted_phone(self, value):
         )
 
     @classmethod
-    def to_raw_phone(cls, value, region=None):
+    def to_raw_phone(self, value, region=None):
```

Comment:
Wasn't this correct when `cls` was used?

Reply:
Fixed.